app/controllers/alchemy/json_api/admin/pages_controller.rb

```diff
@@ -18,6 +18,20 @@ def load_page
         def base_page_scope
           ::Alchemy::Page.all
         end
+
+        def page_scope_with_includes
+          base_page_scope.
+            where(language: Language.current).
+            preload(
+              language: { nodes: [:parent, :page, :children] },
+              draft_version: {
+                elements: [
+                  { page_version: :page },
```

Comment:
I think we don't need to access a page from an element. If an API consumer needs to do that, they have access to the page object - whatever that may be - in their parent scope and can pass it down explicitly.